test/splitEvery.js

```diff
@@ -0,0 +1,34 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('splitEvery', function() {
+
+  it('splits a collection into slices of the specified length', function() {
+    assert.deepEqual(R.splitEvery(1, [1, 2, 3, 4]), [[1], [2], [3], [4]]);
+    assert.deepEqual(R.splitEvery(2, [1, 2, 3, 4]), [[1, 2], [3, 4]]);
+    assert.deepEqual(R.splitEvery(3, [1, 2, 3, 4]), [[1, 2, 3], [4]]);
+    assert.deepEqual(R.splitEvery(4, [1, 2, 3, 4]), [[1, 2, 3, 4]]);
+    assert.deepEqual(R.splitEvery(5, [1, 2, 3, 4]), [[1, 2, 3, 4]]);
+    assert.deepEqual(R.splitEvery(3, []), []);
+  });
+
+  it('dispatches to `slice` method', function() {
```

Comment:
The point to my mind is that unit tests are a form of contract as strong as our type annotations.  They should only document the public behavior of the system.  It's not -- or at least not always -- that we shouldn't implement things based on `slice`, but that we shouldn't commit our public API to that implementation.  It should be no one's business how we decide to do this, and we should feel free to change our minds later.